helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/collectionadapter/ListAdapter.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2022 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.compatibility.collectionadapter;
+
+import com.linkedin.avroutil1.compatibility.StringConverterUtil;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+
+public class ListAdapter extends ArrayList implements List {
```

Comment:
nit - "adapter" sounds like a decorator to me, this copies. call it Transformer maybe?